### ISSUE TITLE
Nefarian gives the wrong class cards

### ISSUE THREAD

**Issue by Patashu** [HearthSim/hs-bugs#59]
If Nefarian is played against a player that had his hero replaced with Ragnaros via Majordomo Executus, 2 spells from the original hero class should be given. Instead, 2 Tail Swipes are currently given instead, the cards that are given when Nefarian is played against Adventure bosses.

https://twitter.com/CM_Zeriyah/status/582679943886209024
https://twitter.com/CM_Zeriyah/status/582678574332653568
http://www.reddit.com/r/hearthstone/comments/34itx3/what_happens_when_you_play_nefarian_against/
https://www.youtube.com/watch?v=w_Cbl41_G-8

If Nefarian is played against an Adventure Boss, 2 Tail Swipes should be given to the player. However, when played against Noth the Plaguebringer, Nefarian gives the player 2 Mage spells. When played against Heigan the Unclean, Nefarian gives the player 2 Warlock spells. This is the case on both Normal and Heroic difficulties.

http://www.reddit.com/r/hearthstone/comments/34p5oa/nefarian_card_interaction_with_bosses_in/

**Comment by jleclanche** [comment]
@Patashu I don't believe this is a bug.

**Comment by Patashu** [comment]
I agree, it's not clear if this is a bug or now the intended behaviour.

**Comment by jleclanche** [comment]
@Patashu We can keep the 2nd part of the bug regarding Heigan.

**Comment by jleclanche** [comment]
BTW I still would like a confirmation before calling this an outright bug. Noth and Heigan can totally be considered warlock/mage.

**Comment by Xinhuan** [comment]
@jleclanche The former striked-out portion you do not believe is a bug, is indeed a bug: https://twitter.com/CM_Zeriyah/status/582679943886209024
https://twitter.com/CM_Zeriyah/status/582698086721437696

The latter is also a bug:
https://twitter.com/CM_Zeriyah/status/583316439215394816
https://twitter.com/CM_Zeriyah/status/582684829570887680

**Comment by troggnostupidhs** [comment]
https://hearthstone.gamepedia.com/Heigan_the_Unclean
https://hearthstone.gamepedia.com/Noth_the_Plaguebringer

![Hearthstone Screenshot 08-26-19 22 27 21](https://user-images.githubusercontent.com/22508641/63736522-5b3ad800-c851-11e9-8e27-c568295665cf.png)
![Hearthstone Screenshot 08-26-19 22 31 49](https://user-images.githubusercontent.com/22508641/63736523-5b3ad800-c851-11e9-960a-b87561d8f73d.png)

AAEBAaIHBrkN1RGBwgLJwgKggAOftwMM+wG9BJsFigb1DfoN8xHLFoStAqvCAo+XA6GhAwA=
